src/main/kotlin/no/nav/syfo/behandler/api/person/BehandlerPersonApi.kt

```diff
@@ -0,0 +1,61 @@
+package no.nav.syfo.behandler.api.person
+
+import io.ktor.application.*
+import io.ktor.http.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import no.nav.syfo.application.api.authentication.personIdent
+import no.nav.syfo.behandler.BehandlerService
+import no.nav.syfo.behandler.api.person.access.PersonAPIConsumerAccessService
+import no.nav.syfo.behandler.domain.hasAnId
+import no.nav.syfo.behandler.domain.toBehandlerDialogmeldingPersonDTO
+import no.nav.syfo.util.getBearerHeader
+import no.nav.syfo.util.getCallId
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+private val log: Logger = LoggerFactory.getLogger("no.nav.syfo")
+
+const val personApiBehandlerPath = "/api/person/v1/behandler"
+const val personBehandlerSelfPath = "/self"
+
+fun Route.registerBehandlerPersonApi(
```

Comment:
Foreslår å bytte navn til PersonBehandlerApi, evt bare BehandlerApi siden det er det den skal returnere. Det det faktisk er gir mer mening at er sist for å klare å kategorisere (som i forskjellen på isbil og bil-is).

Reply:
Kan ikke hat 2 filer som heter  `BehandlerApi`, så mye ha noe mer som sier at API er for person.
Men kan godt bytte til `PersonBehandlerApi`.

Reply:
Endret til BehandlerPerson -> PersonBehandler der det var tilfellet.

Reply:
Er det forresten noen spesiell grunn til at begrepet "Person" er brukt her, i stedet for f.eks. "arbeidstaker", "innbygger" e.l?

Reply:
Ja, fordi personen ikke trenger å være "arbeidstaker". Innbygger har jeg aldri brukt.